clients/fides-js/src/components/ConsentModal.tsx

```diff
@@ -21,6 +21,12 @@ const ConsentModal = ({
 }) => {
   const { container, dialog, title, closeButton } = attributes;
   const { setTrigger } = useEvent();
+  if (!window.Fides) {
```

Comment:
👍 - but ideally shouldn't we be passing this config option down to this component instead of trying to retrieve window.Fides? Don't have to address here, but just curious if you know why we took this approach.